java-util/pom.xml

```diff
@@ -85,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.datasketches</groupId>
```

Comment:
@niketh sorry, wrong link. @gianm suggested that here: https://github.com/druid-io/druid/issues/3892#issuecomment-276434181 and I agreed here: https://github.com/druid-io/druid/issues/3892#issuecomment-276589924